### PR TITLE
Initalize ADR for the repository

### PR DESCRIPTION
**Pull request by trusz**
Rendered version: [1. Record Architecture Decision](https://github.com/openscd/open-scd/blob/caabedfdee3c48820a8e3b4160c4c5b21a4d7b59/docs/adr/0001-record-architecture-decisions.md)

### CODE REVIEW

**Review by danyill — COMMENTED** [review]
LGTM. 

I only didn't approve because I feel we need a "maintainer's approval" and that cannot reasonably be "everyone with admin rights" on this repo which is insufficient to give a architecturally harmonious outcome (recently referred to as the "wild west" :horse_racing: :cowboy_hat_face:)

File: docs/adr/0001-record-architecture-decisions.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+We will follow the decisions recorded in the central organizational 
+repository ([.github](https://github.com/openscd/.github)), 
+and record new repo-specific decisions in this repository.
+
+We write ADRs in the `docs` folder instead of a standard `doc`, 
+because `doc` is used for the generated documentation.
+
+## Consequences
+
+- It will be harder to track which decisions have to be taken into consideration
```
Comment (juancho0202) [inline docs/adr/0001-record-architecture-decisions.md:24]: Wouldn't it be easier if everyone adheres to creating ADRs before making big changes to the project and we track those taken into consideration by looking at active PRs?

**Review by juancho0202 — CHANGES_REQUESTED** [review]

File: docs/adr/0001-record-architecture-decisions.md (hunk 1 above)
Comment (trusz) [reply docs/adr/0001-record-architecture-decisions.md:24]: I'm not sure if I understand your point.
Yes big, architecture relevant changes should be a PR with an ADR
The problem is if we have a global ADR in the .github repo that applies to all the repo and projects in this organisation.

**Review by pascalwilbrink — APPROVED** [review]
LGTM!

File: docs/adr/0001-record-architecture-decisions.md (hunk 1 above)
Comment (trusz) [reply docs/adr/0001-record-architecture-decisions.md]: I agree with your statement generally, but I think that is for another ADR or some other form of agreement, so I think there is no change needed in this PR in that regard.

**Pull request merged**